gammapy/scripts/ts_image.py

```diff
@@ -5,8 +5,9 @@
 import json
 import logging
 log = logging.getLogger(__name__)
-from ..utils.scripts import get_parser
+from ..utils.scripts import get_parser, _configure_root_logger
```

Comment:
Can you use `set_up_logging_from_args` as described [here](https://gammapy.readthedocs.org/en/latest/development/howto.html#configuring-logging-from-command-line-tools) or as used in this example [here](https://github.com/gammapy/gammapy/blob/master/gammapy/scripts/check.py#L34)?
Or is there a reason this doesn't work properly here?